src/FsAutoComplete.Core/CompilerServiceInterface.fs

```diff
@@ -422,14 +422,14 @@ type FSharpCompilerServiceChecker(hasAnalyzers, typecheckCacheSize, parallelRefe
         )
     }
 
-  member __.GetDeclarations(fileName: string<LocalPath>, source, options, _) =
+  member __.GetDeclarations(fileName: string<LocalPath>, source: ISourceText, options: FSharpParsingOptions, _) =
```

Comment:
this is necessary to disambiguate from the new FSharpProjectSnapshot overloads for ParseFile